rules.html

```diff
@@ -122,10 +122,26 @@ <h2>2. Behavior and chat</h2>
             </li>
             <li>
                 <b>2.4 Toxicity.</b><br/>
-                Toxicity is not allowed on this server. Toxicity in chat can be for instance using racial slurs, sexual harassment or severely explicit language.
-                It can also be other non-chat actions which drive players away from the community, for instance portal trapping, luring newly joined players to your town so you can kill them or lock them in your town jail.<br/>
+                Toxicity is not allowed on this server. Toxicity in chat includes but is not limited to:<br/>
+                <ul>

```

Comment:
I don't think there should be an inner list here. The outer list only has two nodes, the inner list and one extra list item added on at the end, doesn't really make much sense structurally.